problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -5,20 +5,26 @@
 		% coordinates \& the standard basis, higher dimensions,
 	\begin{problist}
 		% Computation (4 questions)
-		\prob
+		\prob[\beezer[VO.C10]] 
 		\begin{enumerate}
 			\item
 				Write the following vectors as column vectors.
 				\begin{enumerate}
-					\item $4\xhat -3\zhat +2\yhat -2\xhat\in\R^3$.
-					\item $\yhat +\xhat -5\yhat \in\R^2$.
+					\item $4\sbasis{1} -3\sbasis{3} +2\sbasis{2} -2\sbasis{1}\in\R^3$.
+					\item $1\sbasis{2} +\sbasis{1} -5\sbasis{2} \in\R^2$.
 				\end{enumerate}
 			\item
-				Write the following vectors as linear combinations of $\xhat$, $\yhat$, and
-				$\zhat$.
+				Write the following vectors as linear combinations of the standard
+				basis vectors $\sbasis{1}$, $\sbasis{2}$, $\sbasis{3}$, and
+				$\sbasis{4}$:
 				\begin{enumerate}
-					\item $\mat{1\\-2\\3}$.
-					\item $\mat{-2\\5\\4} + \mat{1\\-2\\ -5} + \mat{1\\0\\1}$.
+					\item $\mat{0\\-3\\2}$
+					\item $\displaystyle
+						\mat{-2\\5\\4} + (-1)\mat{-1\\2\\ 5} + \frac12\mat{2\\0\\2}$
+					\item $\displaystyle
+						2\mat{-1\\-2\\0\\ 5}
+						+1\mat{3\\4\\-2\\ 1}
+						+\left(-\frac13\right)\mat{0\\-3\\0\\ 6}$
```

Comment:
I changed these so they weren't form Beezer. No one else took any problems from those books, so I think it will be easiest just not to take any problems from them at all.